index.html

```diff
@@ -1552,7 +1552,7 @@ <h2>Filey</h2>
 
   <i style="margin-top: 48px; margin-bottom: 8px">Web design by <a href="https://twitter.com/leahlundqvist">Leah</a>.</i>
   <i>
-    <img src="/img/v-logo.png" style="width:15px;position: relative; top: 3px;">
+    <img alt="A solid blue letter V with thick lines and sharp angles" src="/img/v-logo.png" style="width:15px;position: relative; top: 3px;">
```

Comment:
It is not a solid blue line.  It is two-toned.

Reply:
Yeah I hope it is clear what that is supposed to describe.